### PR TITLE
Add support for newish columns

### PR DESCRIPTION
**Pull request by jfy133**
As identified during spring cleaning by comparing our min data/column_elements table and the Pandora CHANGELOG

### CODE REVIEW

**Review by jfy133 — COMMENTED** [review]
Does the 'unknown' table refer to the restricted table file that has been deleted?

**Comment by nevrome** [comment]
Nono - any table that isn't specified by us. This change is supposed to close #64. I thought I could quickly solve a number of small issues with this release

**Comment by jfy133** [comment]
OK could you check why the restricited tables `.Rda` has been deleted in https://github.com/sidora-tools/sidora.core/pull/69/commits/27f297722544855cd14731f877519d1e0bfc6ee2

**Comment by nevrome** [comment]
I think I'm happy with it now :+1:

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+- added a new test to check if `pandora_column_types` and `pandora_table_elements` are in sync
+- added a new test to check if `pandora_column_types` actually specifies all variables in Pandora
+
+## Changes in columns/tables
+
+- TAB_Site: +Date_From, +Date_To, +Date_Info
+- TAB_Individual: +Genetic_Sex, +Archaeological_Date_From, +Archaeological_Date_To, +Archaeological_Date_Info, +Archaeological_Culture, +Archaeological_Period, C14_C:N -> C14_C_N
+- TAB_Extract: +Robot_Name, -Position_on_Plate, +CoreDB_Lysate_Id
+- TAB_Library: +Robot_Name, -Position_on_Plate
+- TAB_Capture: +Robot_Name, -Position_on_Plate, +CoreDB_Pool_Id
```
Comment (jfy133) [inline NEWS.md:15]: Why is position on plate removed? That's still on the web interface :thinking:

**Review by jfy133 — COMMENTED** [review]
Is it expected that prejoin.rda is also is lost?

**Pull request merged**